### DISCUSSION TITLE
本地运行ChatGLM3速度很慢

### DISCUSSION THREAD

**Discussion by ttttddddd** [zai-org/ChatGLM3#1041] [Q&A] [answered]
本地运行ChatGLM3速度很慢 问一个问题要3.4分钟分钟才回答，在网上没查到解决方法，求助各位大佬

**Comment by zkyguo** [comment]
我发现不用量化反而会快很多。但还是特别慢。不知道你找到办法没有？

**Comment by ttttddddd** [comment]
我这最后发现就是没量化 量化之后速度就正常了

---- 回复的原邮件 ----
| 发件人 | Rui ***@***.***> |
| 发送日期 | 2024年05月05日 07:00 |
| 收件人 | THUDM/ChatGLM3 ***@***.***> |
| 抄送人 | Tan ***@***.***>,
Author ***@***.***> |
| 主题 | Re: [THUDM/ChatGLM3] 本地运行ChatGLM3速度很慢 (Discussion #1041) |

我发现不用量化反而会快很多。但还是特别慢。不知道你找到办法没有？

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
如果配置不够，推荐使用量化版本
